internal/controller/utils.go

```diff
@@ -247,14 +247,14 @@ func verifyWorkloadIdentityParameters(workloadIdentity *acpv1.WorkloadIdentityPa
 	var authCount int = 0
 
 	if workloadIdentity.ManagedIdentityClientId != nil {
-		if strings.EqualFold(os.Getenv(WorkloadIdentityDisableGlobalServiceAccount), "true") {
+		if strings.EqualFold(os.Getenv(WorkloadIdentityGlobalServiceAccountEnabled), "false") {
```

Comment:
Why are we reading it from the environment? I thought it's a parameter passed in.

Reply:
The gap is how the controller can know the global service account is enabled or disabled, it just know it via the environment variable that been set while installation.